Client/Application/AppDelegate.swift

```diff
@@ -209,6 +209,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         // Override point for customization after application launch.
         var shouldPerformAdditionalDelegateHandling = true
 
+        // BVC generally handles theme applying, but in some instances views are established
+        // before then (e.g. passcode, so can be privacy concern, meaning this should be called ASAP)
+        // In order to properly apply background and align this with the rest of the UI (keyboard / header)
+        // this needs to be called. UI could be handled internally to view systems,
+        // but then keyboard may misalign with Brave selected theme override
+        Theme.of(nil).applyAppearanceProperties()
```

Comment:
The main thing is ideally this is quite rare. I ma primary hesitant to use a `.default` or `.mode` since it should be so rarely used I don't want to make it super convenient. Otherwise, it may just be used naively.